live-site/components/welcome-section/WelcomeSection.styles.ts

```diff
@@ -8,24 +8,28 @@ const StyledWelcomeSectionContainer = styled.div`
   flex-direction: row;
   justify-content: center;
   margin: 1.3em;
-  @media ${max.tablet} {
+  @media ${max.tabletLg} {
     flex-direction: column;
     align-items: center;
   }
 `;
 
 const StyledRaceContainer = styled.div`
   background-color: ${colors.TEXT_BOX};
-  width: 40em;
-  height: 28.5em;
+  width: 45em;
   border-radius: 1.5em;
-  margin: 1.3em 0 1.3em 1.3em;
+  padding: 3em 2em;
   @media ${max.tablet} {
     margin: 1.3em 0 0 0;
     width: 90%;
   }
 `;
 
+const StyledRaceContent = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
 const StyledWelcomeSectionContent = styled.div`
```

Comment:
Can you change this to be the following so there's a bit more spacing in between the two sections?

```

const StyledWelcomeSectionContent = styled.div`
  display: flex;
  flex-direction: column;
  width: 35.5em;
  margin-right: 2em;
  @media ${max.tabletLg} {
    margin-right: 0;
    margin-bottom: 2em;
    width: 90%;
  }
`;
```